### PR TITLE
Fix new typos

### PR DESCRIPTION
**Pull request by tas50**
Fix typos in https://github.com/mondoohq/cnquery/pull/654 and https://github.com/mondoohq/cnquery/pull/655

Signed-off-by: Tim Smith <tsmith84@gmail.com>

### CODE REVIEW

File: apps/cnquery/cmd/run.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func init() {
 
 var execCmd = builder.NewProviderCommand(builder.CommandOpts{
 	Use:   "run",
-	Short: "Run an MQL query",
+	Short: "Run an MQL query.",
```
Comment (chris-rock) [inline apps/cnquery/cmd/run.go:26]: Should we really have a dot in that short description. It always confuses me.

**Review by chris-rock — COMMENTED** [review]
Thank you

**Review by chris-rock — APPROVED** [review]
@tas50 Thank you for the fix. We can discuss about the final sentence dot in a follow up PR

File: apps/cnquery/cmd/run.go (hunk 1 above)
Comment (tas50) [reply apps/cnquery/cmd/run.go:26]: This was the only one without a period and it looked really off in the main pack docs

**Pull request merged**